### PR TITLE
Add a Helm option to the Getting Started guide.

### PR DESCRIPTION
**Pull request by georgegoh**
For users who require customization of their Contour deployment, we point
them to the Bitnami Helm Chart.

With reference to #2050 - we are pointing users towards a Bitnami hosted helm chart,
as the charts and containers are maintained, hardened and tested by the Bitnami team(see
the following links for hardening):

- https://docs.bitnami.com/tutorials/bitnami-best-practices-hardening-containers/
- https://docs.bitnami.com/tutorials/bitnami-best-practices-hardening-charts/

@youngnick @jpeach 

Signed-off-by: George Goh <gohge@vmware.com>

### CODE REVIEW

**Comment by stevesloka** [comment]
There was talk of adding the helm chart here into the Contour repo. Is that still the plan or is our goal to support the Bitnami chart?

**Comment by georgegoh** [comment]
@stevesloka there was a discussion between @youngnick @jpeach and the Bitnami folks around this topic.

The points discussed of using Bitnami as the official chart:
1. Bitnami pipeline updates containers in a daily builds to ensure that images are secure and updated every day.
2. The charts are maintained by Bitnami's team and tested in their pipeline.
3. New releases of projectcontour are proactively monitored and the Bitnami pipeline automatically triggers a new release of the contour container and chart in cascade when a new version is released.
4. Regarding security, Bitnami follows a very extensive hardening guide
  - https://docs.bitnami.com/tutorials/bitnami-best-practices-hardening-containers/
  - https://docs.bitnami.com/tutorials/bitnami-best-practices-hardening-charts/

**Comment by youngnick** [comment]
Yes, sorry @stevesloka, this was a bit of a side-channel discussion. We've agreed with the Bitnami folks that we will bless their repo instead of bringing it under `projectcontour`.

**Review by youngnick — APPROVED** [review]
LGTM for now, although once the operator is ready, we'll probably reference it as well.

**Comment by stevesloka** [comment]
Hey @georgegoh could you rebase and resolve the conflict now with the `getting-started.md` file? I can merge once that's cleaned up. Thanks!

**Comment by georgegoh** [comment]
@stevesloka done. Please take a look.

File: site/getting-started.md [hunk 1]
```diff
@@ -57,7 +60,21 @@ Deploying Contour using either option creates:
 - A Service of `type: LoadBalancer` that points to the Contour's Envoy instances
 - Depending on your deployment environment, new cloud resources -- for example, a cloud load balancer
 
-### Example workload
+### Option 2: Install using Helm
```
Comment (youngnick) [inline site/getting-started.md:63]: This should be 'Option 3: Install using Helm' now, could you also please add "install using operator" to Option 2 above?

**Review by youngnick — CHANGES_REQUESTED** [review]

File: site/getting-started.md (hunk 1 above)
Comment (georgegoh) [reply site/getting-started.md]: Updated in the latest commit.

**Review by stevesloka — APPROVED** [review]
/lgtm

File: site/getting-started.md [hunk 2]
```diff
@@ -17,15 +17,18 @@ Before you start you will need:
 
 ## Add Contour to your cluster
 
-### Option 1
+### Option 1: Quickstart
 
-Deploy Contour using the rendered manifest:
+As the name suggests, use this option for a quick start with reasonable defaults. If
+you have specific requirements to customize Contour for environment, see Option 2.
```
Comment (skriss) [inline site/getting-started.md:23]: I think this should now be:

```suggestion
you have specific requirements to customize Contour for environment, see Options 2 and 3.
```

**Review by skriss — APPROVED** [review]
one nit, otherwise LGTM

File: site/getting-started.md (hunk 2 above)
Comment (skriss) [reply site/getting-started.md:23]: @georgegoh if you can fix this nit, we should be good to merge. thanks!

**Pull request merged**